rules/explicitOverrideRule.ts

```diff
@@ -74,17 +80,20 @@ export class Rule extends Lint.Rules.TypedRule {
         const hasJsDocParameter = this.ruleArguments.indexOf(OPTION_JSDOC_TAG) !== -1;
         const hasDecoratorParameter = this.ruleArguments.indexOf(OPTION_DECORATOR) !== -1;
         const hasExcludeInterfacesParameter = this.ruleArguments.indexOf(OPTION_EXCLUDE_INTERFACES) !== -1;
+        const hasLowercaseParameter = this.ruleArguments.indexOf(OPTION_NAME_LOWERCASE) !== -1;
+        const hasUppercaseParameter = this.ruleArguments.indexOf(OPTION_NAME_UPPERCASE) !== -1;
```

Comment:
Add an exception if both options are specified.

Reply:
You want me to throw an error here? Just `throw new Error(message)` or something else? I haven't done tslint stuff before, bear with me 😅

Reply:
Maybe it'd be better to just match the boolean functionality with a single argument, rather than keeping the lowercase version `"@override"` that does nothing. The only reason it's really there is to try and mirror the jsdoc & decorator params, but maybe that's silly?

Reply:
That's right, there are in fact only two cases: 'lower', and 'upper'. idk why I've also considered 'any' but that's not a real case.

Then I guess there could be a single parameter, like 'fix-uppercase'. You should document that in the readme too.